react/features/prejoin/functions.js

```diff
@@ -181,6 +181,16 @@ export function getFullDialOutNumber(state: Object): string {
     return `+${country.dialCode}${dialOutNumber}`;
 }
 
+/**
+ * Returns true if a video track other than 'null' was set.
+ *
+ * @param {Object} state - The state of the app.
+ * @returns {boolean}
+ */
+export function isPrejoinVideoTrackSet(state: Object): boolean {
```

Comment:
why no use `getVideoTrack` and check if it's undefined?

Reply:
Done.